#!/usr/bin/env python
"""General DSG visualizer config."""
PACKAGE = "kimera_dsg_visualizer"

import dynamic_reconfigure.parameter_generator_catkin as dr_gen  # NOQA

gen = dr_gen.ParameterGenerator()

gen.add("min_hue", dr_gen.double_t, 0, "min color hue", 0.55, 0.0, 1.0)
gen.add("max_hue", dr_gen.double_t, 0, "max color hue", 0.8, 0.0, 1.0)
gen.add("min_saturation", dr_gen.double_t, 0, "min color saturation", 0.5, 0.0, 1.0)
gen.add("max_saturation", dr_gen.double_t, 0, "max color saturation", 0.7, 0.0, 1.0)
gen.add("min_luminance", dr_gen.double_t, 0, "min color luminance", 0.3, 0.0, 1.0)
gen.add("max_luminance", dr_gen.double_t, 0, "max color luminance", 0.8, 0.0, 1.0)

exit(gen.generate(PACKAGE, "kimera_scene_graph", "Colormap"))
